src/bin/tikv-server.rs

```diff
@@ -292,6 +312,13 @@ fn main() {
                 .value_name("IP:PORT")
                 .help("Sets advertise listening address for client communication"),
         )
+        .arg(
+            Arg::with_name("status-addr")
+                .long("status-addr")
+                .takes_value(true)
+                .value_name("IP:PORT")
+                .help("Sets status report HTTP service address"),
```

Comment:
Please make it align with previous item description, i.e. Sets HTTP listening address for the status report service.